modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -117,11 +121,3 @@ $ oc apply -f sub.yaml
 At this point, the OLM is now aware of the selected Operator. A
 ClusterServiceVersion (CSV) for the Operator should appear in the target
 namespace, and APIs provided by the Operator should be available for creation.
-
-.Additional resources
-
-* To install custom Operators to a cluster using the OperatorHub, you must first
-upload your Operator artifacts to Quay.io, then add your own `OperatorSource` to
-your cluster. Optionally, you can add Secrets to your Operator to provide
-authentication. After, you can manage the Operator in your cluster as you would
-any other Operator. For these steps, see link:https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md[Testing Operators].
```

Comment:
@ecordell While working in this topic, noticed this section at the end that I think can be removed now that we can create custom catalog images. WDYT? @chrisnegus

Reply:
Ack'd with Chris.